app/models/like.rb

```diff
@@ -1,4 +1,14 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_likes_counter
+    like_counter = post.likes_counter
+    if like_counter.nil?
+      like_counter = 1
+    else
+      like_counter += 1
+    end
+    post.update(likes_counter: like_counter)
+  end
```

Comment:
[OPTIONAL]
```suggestion
  after_save :update_likes_counter
  
  private
  
  def update_likes_counter
    like_counter = post.likes_counter
    if like_counter.nil?
      like_counter = 1
    else
      like_counter += 1
    end
    post.update(likes_counter: like_counter)
  end
```
- [ ] Nice work implementing the `update_likes_counter` method :clap: However it would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new post has been saved to database, then make the `update_likes_counter` method private